imports/startup/client/routes/projects.js

```diff
@@ -40,11 +40,6 @@ Router.route(
         return Roles.userIsInRole(Meteor.userId(), DefaultAdminAccessRoles);
       },
     },
-    waitOn() {
-      return [
-        Meteor.subscribe('projects.all'),
```

Comment:
Why did you remove projects subscription?

Reply:
There is no need for subscription on create project page and after removing subscription everything looks working fine.